src/main/menu_main.cpp

```diff
@@ -1344,7 +1344,7 @@ bool mainMenuUpdate()
         else if(MenuMode == MENU_SELECT_SLOT_1P || MenuMode == MENU_SELECT_SLOT_2P)
         {
             if(SharedCursor.Move)
-                s_handleMouseMove(4, MenuX, MenuY, 300, 30);
+                s_handleMouseMove(c_menuItemSavesDelete, MenuX, MenuY - c_menuSavesOffsetY, 300, 30);
```

Comment:
Multires sets MenuY to 30 on very low res screens. Instead of adding menu offset Y to MenuY, use it in GetMenuPos when calculating MenuY for the save select menu modes.